crates/sui-framework/packages/sui-framework/sources/pay.move

```diff
@@ -16,23 +16,19 @@ public fun keep<T>(c: Coin<T>, ctx: &TxContext) {
 }
 
 /// Split coin `self` to two coins, one with balance `split_amount`,
-/// and the remaining balance is left is `self`.
+/// and the remaining balance is left in `self`.
```

Comment:
Amusingly, there is no `self` coin in the parameter names. So we should either rename the parameter or change the comment